pkgs/by-name/xd/xdg-terminal-exec/package.nix

```
@@ -0,0 +1,35 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+}:
+stdenv.mkDerivation {
+  pname = "xdg-terminal-exec";
+  version = "unstable-2023-12-08";
+
+  src = fetchFromGitHub {
+    owner = "Vladimir-csp";
+    repo = "xdg-terminal-exec";
+    rev = "04f37d4337b6ce157d4a7338dd600a32deb43a28";
+    hash = "sha256-QIPdF+/dMUEVcz5j9o+wQ4dnw2yWwz7slnLdMNETkGs=";
+  };
+
+  dontBuild = true;
+  installPhase = ''
+    runHook preInstall
+
+    mkdir -p $out/bin
+    cp xdg-terminal-exec $out/bin
```

```suggestion
    install -Dm555 xdg-terminal-exec -t $out/bin
```
and the `mkdir` line can be removed